tests/PasswordHasherTest.php

```diff
@@ -27,4 +27,13 @@ public function testValidateEmptyPasswordException()
         $password = new PasswordHasher();
         $password->validate('', 'test');
     }
+
+    /**
+     * In PHP 7.4 password hashing algorithm identifiers are now nullable strings rather than integers.
+     */
+    public function testAlgorithmString()
+    {
+        $password = new PasswordHasher('test');
+        $this->assertTrue(true);
```

Comment:
Maybe it's better to assert that string is returned?

Reply:
Returned from what?

Reply:
It's OK as is.